src/dimension/MapDimension.js

```diff
@@ -58,12 +58,29 @@ WGL.dimension.MapDimension = function(id){
 			}
 		}
 		/*set point size*/		
-	//	console.log( map.getZoom());
+	    //console.log( map.getZoom());
 		
-		gl.uniform1f(this.glProgram.loc, manager.zoom);				
+		//gl.uniform1f(this.glProgram.loc, manager.zoom);
+
+		var pointsizeLoc = gl.getUniformLocation(this.glProgram, "pointsize");
+		gl.uniform1f(pointsizeLoc, this.pointSize(manager.zoom));
```

Comment:
Slo by tohle udelat radeji tak jak je to v HeatMapDimension? Tzn na zacatku ulozit location jako 	
var psize = "psize"
manager.storeUniformLoc(this.glProgram, psize);
 a pak  pri renderovani:
gl.uniform1f(this.glProgram[psize], hodnotapsize);